fmlcore/src/main/java/net/minecraftforge/fml/OptionalMod.java

```diff
@@ -129,7 +129,7 @@ public void ifPresent(Consumer<? super T> consumer) {
      * and if the result is non-null, return an {@code Optional} describing the
      * result.  Otherwise return an empty {@code Optional}.
      *
-     * @apiNote This method supports post-processing on optional values, without
+     * API Note: This method supports post-processing on optional values, without
```

Comment:
What's the point in doing this if you added support for the annotation? Also, if continuing to do this, should add `<p>`.

Reply:
This change was done before I re-added support for those JDK-specific annoatations (after I realized that those were used a lot more often than I thought). I'll be reverting that.